packages/arb-token-bridge-ui/src/hooks/useNetworks.ts

```diff
@@ -82,10 +82,34 @@ export function sanitizeQueryParams({
     isSupportedChainId(sourceChainId) &&
     !isSupportedChainId(destinationChainId)
   ) {
-    const [defaultDestinationChainId] = getDestinationChainIds(sourceChainId)
-    return {
-      sourceChainId: sourceChainId,
-      destinationChainId: defaultDestinationChainId!
+    switch (sourceChainId) {
```

Comment:
Chains are sorted by chain ID in descending order. In L97 we grab first chain, if that chain's ID is lower than Arbitrum One's or Arbitrum Sepolia's chain, it will default to that chain instead. This change makes sure we always default to our core chains instead an Orbit chain.